docs/administration/deployment/upgrading-db/index.md

```diff
@@ -0,0 +1,33 @@
+# Upgrading database
+
+There might be changes in the database schema between some REANA
+versions. In these cases, to ensure consistency, it is necessary to run
+a migration script using [`alembic`](https://alembic.sqlalchemy.org/en/latest/).
+
+The `reana-db` CLI wraps this functionality to ease such migrations
+after upgrading your REANA instance.
+
+After running `helm upgrade` successfully, run the following command:
+
+```console
+$ kubectl -n <reana_namespace> exec -i -t deployment/<reana_release_name>-server -- reana-db alembic upgrade
+```
+
+The output will show the upgrades accomplished.
+
+Note that is needed to replace the placeholders (`<reana_namespace>` and
+`<reana_release_name>`) appropriately. For instance, for namespace
+`default` and release name `reana`, this is the command and a possible
+output:
+
+```console
+$ kubectl -n default exec -i -t deployment/reana-server -- reana-db alembic upgrade
+Defaulted container "rest-api" out of: rest-api, scheduler
+INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+INFO  [alembic.runtime.migration] Will assume transactional DDL.
+INFO  [alembic.runtime.migration] Running upgrade  -> c912d4f1e1cc, Quota tables.
+INFO  [alembic.runtime.migration] Running upgrade c912d4f1e1cc -> ad93dae04483, Interactive sessions.
+INFO  [alembic.runtime.migration] Running upgrade ad93dae04483 -> 4801b98f6408, Job started and finished times.
+INFO  [alembic.runtime.migration] Running upgrade 4801b98f6408 -> f84e17bd6b18, Workflow complexity.
+INFO  [alembic.runtime.migration] Running upgrade f84e17bd6b18 -> 6568d7cb6710, storing full workflow workspace.
+```
```

Comment:
It would be good to add two new paragraphs with two more examples showcasing:

- `alembic history` so that people can see what has been applied previously on their systems;
- `alembic downgrade` so that people can go back in case something goes wrong with the deployment.

This may be useful for cluster administrators that are not familiar with Alembic.

P.S. And we could perhaps end up with referring people to the help:

```console
$ kubectl exec deployment/reana-server -c rest-api -- reana-db alembic --help 
```

Reply:
Added a user story following these steps:
1. upgrade
2. history
3. current
4. downgrade
5. current